webpack/envs/base.js

```diff
@@ -70,6 +73,24 @@ export const baseConfig = {
       jade: "jade/runtime.js",
       waypoints: "jquery-waypoints/waypoints.js",
     }),
+    new DuplicatesPlugin({
+      verbose: true,
+      emitHandler(report) {
+        const artifactsPath = path.join(process.cwd(), ".artifacts")
+        fs
+          .mkdir(artifactsPath)
+          .catch(() => Promise.resolve()) // .artifact directory exists, continue...
+          .then(() =>
+            fs.writeFile(
+              path.join(artifactsPath, "duplicates-report"),
+              stripAnsi(report)
+            )
+          )
+          .catch(err => {
+            console.error("Could not write duplicates report", err)
```

Comment:
console.error(`[DuplicatesPlugin] Could not write duplicates report, err`)

Reply:
```suggestion
            console.error("[DuplicatesPlugin] Could not write duplicates report", err)
```